src/start/IosCompliantAppleAuthButton/Custom.js

```diff
@@ -23,7 +23,7 @@ const styles = createStyleSheet({
     borderRadius: 22,
     overflow: 'hidden',
   },
-  nightFrame: {
+  darkFrame: {
     backgroundColor: 'black',
   },
   dayFrame: {
```

Comment:
This should change from "day" to "light", to match the change from "night" to "dark".

Same for the other mention of "day" in this file.